lib/ts/recipe/session/types.ts

```diff
@@ -224,32 +222,36 @@ export type RecipeInterface = {
      *
      * Returns undefined if the sessionHandle does not exist
      */
-    getSessionInformation(input: {
-        sessionHandle: string;
+    getSessionInformation(input: { sessionHandle: string; userContext: any }): Promise<SessionInformation | undefined>;
+
+    revokeAllSessionsForUser(input: {
+        userId: string;
         tenantId: string;
+        revokeAcrossAllTenants?: boolean;
         userContext: any;
-    }): Promise<SessionInformation | undefined>;
-
-    revokeAllSessionsForUser(input: { userId: string; tenantId: string; userContext: any }): Promise<string[]>;
+    }): Promise<string[]>;
 
-    getAllSessionHandlesForUser(input: { userId: string; tenantId: string; userContext: any }): Promise<string[]>;
+    getAllSessionHandlesForUser(input: {
+        userId: string;
+        tenantId: string;
+        fetchAcrossAllTenants?: boolean;
+        userContext: any;
+    }): Promise<string[]>;
 
-    revokeSession(input: { sessionHandle: string; tenantId: string; userContext: any }): Promise<boolean>;
+    revokeSession(input: { sessionHandle: string; userContext: any }): Promise<boolean>;
 
-    revokeMultipleSessions(input: { sessionHandles: string[]; tenantId: string; userContext: any }): Promise<string[]>;
+    revokeMultipleSessions(input: { sessionHandles: string[]; userContext: any }): Promise<string[]>;
 
```

Comment:
SessionInformation type should also contain a tenantId

Reply:
Done